src/index.js

```diff
@@ -1,9 +1,38 @@
 import angular from 'angular';
 import uiRouter from 'angular-ui-router';
 
-angular
-  .module('spotifyzier', [uiRouter])
-  .config(($urlRouterProvider) => {
-    'ngInject';
+import SpotifyService from './service'
+import { AppSearchComponent } from './app-search.component'
+import { AppSearchResultComponent } from './app-search-result.component'
+import { AppAlbumComponent } from './app-album.component'
+import { AppAlbumCommentsComponent } from './app-album-comments.component'
+import { AppCommentComponent } from './app-comment.component'
+import { AppCommentFormComponent } from './app-comment-form.component'
+
+const root = angular
+  .module('spotifyzier', [
+    uiRouter
+  ])
+  .service('SpotifyService', SpotifyService)
+  .component('search', AppSearchComponent)
```

Comment:
Every component should have his module, to allow to smart use the dependency injection.